diff --git a/node_modules/openrgb-sdk/types/client.d.ts b/node_modules/openrgb-sdk/types/client.d.ts
index 34ca346..9295f6d 100644
--- a/node_modules/openrgb-sdk/types/client.d.ts
+++ b/node_modules/openrgb-sdk/types/client.d.ts
@@ -44,7 +44,7 @@ declare class Client {
      * get the properties of all devices
      * @returns {Promise<Device>[]}
      */
-    getAllControllerData(): Promise<Device>[];
+    getAllControllerData(): Promise<Device[]>;
     /**
      * get a list of all profiles
      * @returns {Promise<String>[]}
diff --git a/node_modules/openrgb-sdk/types/device.d.ts b/node_modules/openrgb-sdk/types/device.d.ts
index 41c37c7..ea97651 100644
--- a/node_modules/openrgb-sdk/types/device.d.ts
+++ b/node_modules/openrgb-sdk/types/device.d.ts
@@ -3,6 +3,7 @@ declare class Device {
     constructor(buffer: any, deviceId: any, protocolVersion: any);
     deviceId: any;
     type: any;
+    name: string;
     activeMode: any;
     modes: {
         id: number;
